ragmac_xdem/dem_postprocessing.py

```diff
@@ -74,7 +75,14 @@ def calculate_init_stats_single(
     :returns: a tuple containing - basename of DEM, path to DEM, count of obs, median and NMAD over stable terrain, coverage over roi
     """
     # Load DEM and reproject to ref grid
+    ref_dem = xdem.DEM(ref_dem)
     dem = xdem.DEM(dem_path)
+    common_bounds = gu.projtools.merge_bounds([ref_dem.bounds,dem.bounds],merging_algorithm='intersection')
```

Comment:
Actually, if you want to calculate the intersection between two rasters only, you can do so with `raster1.intersection(raster2)`. The merge_bounds function is more meant for a list of rasters.

Reply:
I also see that this approach can reduce memory requirement, but won't it increase processing time? Each time the reference DEM needs to be loaded in memory again, and reprojected, which could take several (tens of?) seconds for large DEMs.